src/librustc_borrowck/diagnostics.rs

```diff
@@ -198,7 +198,7 @@ fn main() {
 ```
 "##,
 
-E0386: r##"
+/*E0386: r##"
```

Comment:
We put these into `register_diagnostics! {}`, commented out, although a better way to annotate codes that may not be reused should be made.